Mathlib/RingTheory/Ideal/Span.lean

```diff
@@ -124,6 +124,16 @@ theorem span_singleton_ne_top {α : Type*} [CommSemiring α] {x : α} (hx : ¬Is
 @[simp]
 theorem span_zero : span (0 : Set α) = ⊥ := by rw [← Set.singleton_zero, span_singleton_eq_bot]
 
+@[simp]
+lemma span_singleton_zero : span {(0 : α)} = ⊥ := Submodule.span_zero_singleton _
+
+@[simp]
+lemma span_insert_zero {s : Set α} : span (insert 0 s) = span s := Submodule.span_insert_zero
```

Comment:
Another option would be to tag `span_insert` as a simp-lemma? (Otherwise you start needing to add more simp-lemmas like `span_insert_one`)

Reply:
In my opinion, assuming  `span R {x} ⊔ span R s` seen as normal form, `span R s ⊔ span R t` should also been seen as normal form of `span R (s ∪ t)`, which implies that `span_union : span R (s ∪ t) = span R s ⊔ span R t` should also be marked as `@[simp]`. Both of them are beyond this PR.